packages/plugin-ext/src/common/rpc-protocol.ts

```diff
@@ -149,6 +154,39 @@ export class RPCProtocolImpl implements RPCProtocol {
     }
 }
 
+export class InitializationCallbackImpl implements ProxySynchronizer {
```

Comment:
Could we maybe calls this class `DefaultProxySynchronizer` or `ProxySynchronizerImpl` or something similar instead of `InitializationCallbackImpl` since I think the name `callback` is not that great in that context. That also applies to the property name then.

Reply:
Sorry, missed this one during the rename